### DISCUSSION TITLE
Are Polls a viable substitute for Buttons?

### DISCUSSION THREAD

**Discussion by TitoBahe** [devlikeapro/waha#644] [Q&A] [answered]
Why are Buttons so unstable? 
Are Polls more stable than them?  

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment, marked as answer]
> Why are Buttons so unstable?

WA makes it hard to send it, always comes up with smthg to hide it :( It's not in any official clients (web/android/ios) and only available in Business API. So if you really need it - only this way to go

> Are Polls more stable than them?

Yes! Everything that available in regular Web WhatsApp is pretty stable.

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
